FinSearchUnified/Resources/config.xml

```diff
@@ -27,6 +27,18 @@
             <description lang="de">Aktiviert die FINDOLOGIC Suche für Kategorieseiten.</description>
             <description lang="en">Activate the FINDOLOGIC search provider for category pages.</description>
         </element>
+        <element type="combo">
+            <name>CrossSellingCategories</name>
```

Comment:
Please ensure that this works with all Shopware versions that we support. Shopware 5.2 should work as well.